spinalcordtoolbox/types.py

```diff
@@ -212,9 +215,32 @@ class Centerline:
                            'L1': 31.89272719870084, 'L2': 33.511890474486449, 'L3': 35.721413718617441}
 
-    """
-    {'T10': ['T10', 25.543101799896391, 2.0015883550878457], 'T11': ['T11', 27.192970855618441, 1.9996136135271434], 'T12': ['T12', 29.559890137292335, 2.0204112073304121], 'PMG': ['PMG', 12.429867526011929, 2.9899172582983007], 'C3': ['C3', 18.229087873095388, 1.3299710200291315], 'C2': ['C2', 18.859365127066937, 1.5764843286826156], 'C1': ['C1', 0.0, 0.0], 'C7': ['C7', 15.543004729447034, 1.5597730786882851], 'C6': ['C6', 15.967482996580138, 1.4698898678270345], 'PMJ': ['PMJ', 11.38265467206886, 1.5641456310519117], 'C4': ['C4', 17.486130819790912, 1.5888243108648978], 'T8': ['T8', 25.649136105105754, 4.6835454011234718], 'T9': ['T9', 25.581999112288241, 1.9565018840832449], 'T6': ['T6', 23.539740893750668, 1.9073272889977211], 'T7': ['T7', 24.388589291326571, 1.828160893366733], 'T4': ['T4', 22.076131620822075, 1.726133989579701], 'T5': ['T5', 22.402770293433733, 2.0157113843189087], 'T2': ['T2', 19.800131846755267, 1.7600195442391204], 'T3': ['T3', 21.287064228802027, 1.8123109081532691], 'T1': ['T1', 16.525065003339993, 1.6130238001641826], 'L2': ['L2', 34.382300279977912, 2.378543023223767], 'L3': ['L3', 34.804841812064133, 2.7878124182683481], 'L1': ['L1', 32.02934490161379, 2.7697447948338381], 'C5': ['C5', 16.878263370935201, 1.6952832966782569]}
-    """
+    # {
+    #     "T10": ["T10", 25.543101799896391, 2.0015883550878457],
+    #     "T11": ["T11", 27.192970855618441, 1.9996136135271434],
+    #     "T12": ["T12", 29.559890137292335, 2.0204112073304121],
+    #     "PMG": ["PMG", 12.429867526011929, 2.9899172582983007],
+    #     "C3": ["C3", 18.229087873095388, 1.3299710200291315],
+    #     "C2": ["C2", 18.859365127066937, 1.5764843286826156],
+    #     "C1": ["C1", 0.0, 0.0],
+    #     "C7": ["C7", 15.543004729447034, 1.5597730786882851],
+    #     "C6": ["C6", 15.967482996580138, 1.4698898678270345],
+    #     "PMJ": ["PMJ", 11.38265467206886, 1.5641456310519117],
+    #     "C4": ["C4", 17.486130819790912, 1.5888243108648978],
+    #     "T8": ["T8", 25.649136105105754, 4.6835454011234718],
+    #     "T9": ["T9", 25.581999112288241, 1.9565018840832449],
+    #     "T6": ["T6", 23.539740893750668, 1.9073272889977211],
+    #     "T7": ["T7", 24.388589291326571, 1.828160893366733],
+    #     "T4": ["T4", 22.076131620822075, 1.726133989579701],
+    #     "T5": ["T5", 22.402770293433733, 2.0157113843189087],
+    #     "T2": ["T2", 19.800131846755267, 1.7600195442391204],
+    #     "T3": ["T3", 21.287064228802027, 1.8123109081532691],
+    #     "T1": ["T1", 16.525065003339993, 1.6130238001641826],
+    #     "L2": ["L2", 34.382300279977912, 2.378543023223767],
+    #     "L3": ["L3", 34.804841812064133, 2.7878124182683481],
+    #     "L1": ["L1", 32.02934490161379, 2.7697447948338381],
+    #     "C5": ["C5", 16.878263370935201, 1.6952832966782569],
+    # }
```

Comment:
Technically I changed this from a string to a comment, but the string was just... created and then dropped? And it's not a docstring? So I'm not sure why it was this way.

Also, as part of wrapping this very long line, I used _Black_, which inserted a trailing comma in the list. If this string value was intended to be Python code that should be fine, but note that JSON does _not_ allow trailing commas.